HLTriggerOffline/Egamma/python/HLTmultiTrackValidator_cff.py

```diff
@@ -0,0 +1,23 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.RecoTrack.HLTmultiTrackValidator_cfi import *
+hltTrackValidator = hltMultiTrackValidator.clone(
+    label = [
+        "hltPixelTracks",
+        "hltIter0PFlowTrackSelectionHighPurity",
+        "hltIter1PFlowTrackSelectionHighPurity",
+        "hltIter1Merged",
+        "hltIter2PFlowTrackSelectionHighPurity",
+        "hltIter2Merged",
+#        "hltIter3PFlowTrackSelectionHighPurity",
+#        "hltIter3Merged",
+#        "hltIter4PFlowTrackSelectionHighPurity",
+#        "hltIter4Merged",
+    ]
+)
+
+hltMultiTrackValidation = cms.Sequence(
+    hltTPClusterProducer
+    + hltTrackAssociatorByHits
+    + hltTrackValidator
+)    
```

Comment:
This file is exactly the same as https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Validation/RecoTrack/python/HLTmultiTrackValidator_cff.py. Why another copy would be needed?